crates/db_views/src/post_view.rs

```diff
@@ -386,6 +386,22 @@ fn queries<'a>() -> Queries<
 
     let (limit, offset) = limit_and_offset(options.page, options.limit)?;
 
+    // when_before and when_after to filter posts by specified published date.
+    // ToDo: establish default convention, likley specific to the sort choice picked
```

Comment:
all caps TODO , and also add another todo that this will need to be updated when #3496 is merged.

Reply:
Implemented, committed